.gitignore

```diff
@@ -3,7 +3,6 @@
 build/
 out/
 
-gradlew
 gradlew.bat
```

Comment:
Should probably commit `gradle.bat` also so windows isn't left out with being able to have a gradle wrapper available.

Reply:
Fair point! fixing this 😄